### PR TITLE
app-text/lesspipe: Enable epatch_user

### PR DESCRIPTION
**Pull request by asgeir**
Package-Manager: Portage-2.3.24, Repoman-2.3.6

### CODE REVIEW

**Comment by Polynomial-C** [comment]
Would you mind re-doing this without the forth- and backwards change in the stable ebuild? Your EAPI-6 ebuild looks good.

**Comment by asgeir** [comment]
@Polynomial-C Should I close this pull request then and redo it with just the correct changes?

**Comment by Polynomial-C** [comment]
Whatever you like better. I persoanlly would revert the changes in my local git branch, do a proper commit and push -f to my github repo.

File: app-text/lesspipe/lesspipe-1.83-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+src_compile() {
+	# Nothing to build.
+	:
+}
+
+src_install() {
+	emake PREFIX="${EPREFIX}/usr" DESTDIR="${D}" install
+	dodoc ChangeLog README TODO
```
Comment (a17r) [inline app-text/lesspipe/lesspipe-1.83-r1.ebuild]: You could replace this line with simply `einstalldocs`

**Pull request closed without merging**